docker/src/main/java/brooklyn/entity/container/docker/repository/DockerRepositoryDriver.java

```diff
@@ -0,0 +1,6 @@
+package brooklyn.entity.container.docker.repository;
+
+import brooklyn.entity.container.docker.application.VanillaDockerApplicationDriver;
+
+public interface DockerRepositoryDriver extends VanillaDockerApplicationDriver {
```

Comment:
Can we actually just reference the `VanillaDockerApplicationDriver` from our `DockerRepository` entity directly, without the need to create these subclasses?